local/http/http.go

```diff
@@ -212,6 +213,10 @@ func (s *Server) ProxyHandler(w http.ResponseWriter, r *http.Request) {
 
 // Handler handles HTTP requests
 func (s *Server) Handler(w http.ResponseWriter, r *http.Request) {
+	if s.AllowCORS {
+		w.Header().Add("Access-Control-Allow-Origin", "*")
```

Comment:
to make it actually useful on my side I had to add more headers:
```
if origin := r.Header.Get("origin"); origin != "" {
                w.Header().Set("Access-Control-Allow-Origin", origin)
                w.Header().Set("Access-Control-Allow-Methods", "GET, POST, OPTIONS")
                w.Header().Set("Access-Control-Allow-Headers", "DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range,If-None-Match")
                w.Header().Set("Access-Control-Max-Age", "3600")
                w.Header().Set("Access-Control-Allow-Credentials", "true")
            }
```

this is actually why I didn't submit any PR yet it's because I was not satisfied with this and didn't see a clean why to do this.
anyone got some feedbacks?

Reply:
Maybe this could be configurable from the `.symfony.local.yaml` or another file?

Reply:
Sorry for the delay @Kocal 😔
Actually, I didn't use the CLI on this project and probably won't for a while.
So I guess we can move forward!